front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -109,23 +109,37 @@ export const CommandMenu = () => {
         onValueChange={setSearch}
       />
       <StyledList>
-        <StyledEmpty>No results found.</StyledEmpty>
-        {!matchingCreateCommand && (
-          <StyledGroup heading="Create">
-            {commandMenuCommands
-              .filter((cmd) => cmd.type === CommandType.Create)
-              .map((cmd) => (
-                <CommandMenuItem
-                  key={cmd.label}
-                  to={cmd.to}
-                  label={cmd.label}
-                  Icon={cmd.Icon}
-                  shortcuts={cmd.shortcuts || []}
-                  onClick={cmd.onCommandClick}
-                />
-              ))}
-          </StyledGroup>
-        )}
+        {!matchingCreateCommand &&
+          !matchingNavigateCommand &&
+          commandMenuCommands.filter((cmd) => cmd.type === CommandType.Create)
```

Comment:
this looks to be the same check as machingCreateCommand, could you double check?